Registry/registry.polrad

```diff
@@ -0,0 +1,61 @@
+
+rconfig   integer sbm_diagnostics          namelist,physics         max_domains    0       rh    "sbm_opt"            "SBM diagnostic opeion, 1: on"
+
+
+# sbm radar variables
+state   real    -                 ikjf sbmradar      1         -     -   -
+state   real    Water_zh            ikjf  sbmradar        1         -    rh03   "Water_zh"    "Water Horizontal Refl."      "dBZ"
+state   real    Water_zv            ikjf  sbmradar        1         -    rh03   "Water_zv"    "Water Vertical Refl."      "dBZ"
+state   real    Water_zdr           ikjf  sbmradar        1         -    rh03   "Water_zdr"    "Water Differential Refl."      "dBZ"
+state   real    Water_ldr           ikjf  sbmradar        1         -    rh03   "Water_ldr"    "Water Linear Differntial Refl."      "dB"
+state   real    Water_kdp            ikjf   sbmradar        1         -    rh03   "Water_kdp"    " Water KDP  "      "dBZ"
```

Comment:
@JS-WRF-SBM 
Kobby,
Conventionally, we do not have leading and trailing spaces in the description (you have several occurrences of this).